src/Microsoft.AspNetCore.Razor/Editor/ImplicitExpressionEditHandler.cs

```diff
@@ -85,6 +85,11 @@ protected override PartialParseResult CanAcceptChange(Span target, TextChange no
                 return HandleDotlessCommitInsertion(target);
```

Comment:
Some of the inbetween states of Roslyn's new dotless commits are handled by the existing dotless commit code; therefore, these are still necessary.